scripts/detect_artefacts.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+
+import os
+
+os.environ['USE_TORCH'] = '1'
+
+import argparse
+import logging
+
+import cv2
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import figure
+import torch
+
+from doctr.models import obj_detection
+from doctr.io.image import read_img_as_tensor
+
+
+def plot_predictions(image, tg_boxes, tg_labels, cl_map, cm):
+    for ind_2, val_2 in enumerate(tg_boxes):
```

Comment:
```python
for box, label in zip(tg_boxes, tg_labels):
```

Reply:
and then replace all "val_2...." and "tg_labels[..." afterwards

Reply:
alright :)